ruma-events-macros/src/event_enum.rs

```diff
@@ -97,10 +105,58 @@ pub fn expand_stripped_enum(input: &EventEnumInput) -> syn::Result<TokenStream>
     expand_any_enum_with_deserialize(input, &ident)
 }
 
+/// Generates the 3 redacted state enums, 2 redacted message enums,
+/// and `Deserialize` implementations.
+///
+/// No content enums are generated since only five state variants contain content.
```

Comment:
And more importantly, since there's just no part of the API that deals with "any redacted event's content".